how-to/how-to-add-telerik-ui-for-nativescript-package.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to Include the Telerik UI for NativeScript Package
+description: Tutorial on how to include the Telerik UI for NativeScript package to AppBuilder project
+type: how-to
+page_title: How to add the Telerik UI for NativeScript package to an AppBuilder project
+slug: include-nativescript-ui-package
+position:
+tags: appbuilder, nativescript
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package management</td>
+  </tr>
+  <tr>
+    <td>Mobile Development type</td>
+    <td>Native (NativeScript)</td>
+  </tr>
+</table>
+
+## Description
+This article provides the general steps required to include the Telerik UI for NativeScript package in your project.
```

Comment:
Personally, I dislike this type of description as I don't feel that it brings anything to the table. Generally, in technical documentation, descriptions should focus on a particular task or context and should avoid the "This article describes repeat-the-title-here" model.

Think about the task that the user is after. In this case, they want to make use of the Telerik UI for Nativescript package which is available in two editions - a free edition and a pro edition. Think about their context, if any. Provide the general overview of the task to be achieved here. 

Here's an example of how I'd go about writing this description:

Telerik provides [Telerik UI for NativeScript](http://docs.telerik.com/devtools/nativescript-ui/introduction) which lets you incorporate native UI components in your NativeScript apps. To be able to develop with Telerik UI for NativeScript in the Telerik Platform, you need to add the required package in your app in AppBuilder. You can choose between a free limited edition or a complete edition available with specific subscription plans.